src/passes/MinifyImportsAndExports.cpp

```diff
@@ -145,7 +145,7 @@ struct MinifyImportsAndExports : public Pass {
     std::map<Name, Name> oldToNew;
     auto process = [&](Name& name) {
       // do not minifiy special imports, they must always exist
-      if (name == MEMORY_BASE || name == TABLE_BASE) {
+      if (name == MEMORY_BASE || name == TABLE_BASE || name == STACK_POINTER) {
```

Comment:
are these "ABI imports" documented in tool-conventions? if so a link to there might be nice.

for the first two I think the dynamic linking doc there mentions them.

Reply:
I don't know .. all i know is that if I allow minification of __stack_pointer I get a test failure.  To be honest I'm not sure why these can't be minified too.

Reply:
Oh you are correct this should be documented in https://github.com/WebAssembly/tool-conventions/blob/master/DynamicLinking.md but isn't yet.